config/config.go

```diff
@@ -27,7 +27,6 @@ var (
 
 // Config contains all the configuration needed.
 type Config struct {
-	Name string
```

Comment:
Don't we need this to allow multiple Engine to run on the same docker?

Reply:
Nope, we pass this to service, but in fact, we used $MESG_NAME to have a different docker service name. This one was constant all the time.

Reply:
Retested, working fine!

Reply:
What about the address of the Engine? It is used to create the ENDPOINT env in the services.

Reply:
this should not be removed, this is the part that is inserted in the docker service. Now based on the rest of the pull request you inject this `version.Name` but there is no way to change this name where on the config we can change it with `MESG_NAME` (`envconfig.MustProcess(envPrefix, c)`)

With this change we cannot run multiple engine on the same computer. Of course run multiple instances will work but when we will start services then we will have a problem. I don't see any reason to change that behavior.

Reply:
This makes sens , i've reverted the commit + update a branch